src/Shared/Configuration/Files/World.cs

```diff
@@ -37,6 +37,22 @@ public class WorldConfFile : ConfFile
 		public bool BlueOrbFollowWarp { get; protected set; }
 		public bool BlueOrbPetSystem { get; protected set; }
 
+		public float BlueJackpotSpawnChance { get; protected set; }
+		public float BlueJackpotExpRate { get; protected set; }
+		public float SilverJackpotSpawnChance { get; protected set; }
+		public int SilverJackpotRolls { get; protected set; }
```

Comment:
What are "rolls" in this context? That seems like a rather unusual term.

Reply:
A roll is one pass through the the drop list.  So a Silver Monster iterates through its drop list 100 times (by default).  I'm open to another term for it if you have any suggestions, I couldn't come up with anything else that sounded better.

Reply:
Personally I like multiplier better, simply because it's easier to understand, even if it wouldn't be as accurate. But yea, I don't have a suggestion for an actual, objectively better name. And once you know what it actually does, roll makes sense. Unfortunately most users will probably never know how it works^^"

Reply:
Well, if it helps, I put more detail about what it means into the conf file.  I think it's at least reasonably clear what it does now.